doc/api-extensions.md

```diff
@@ -2520,3 +2520,7 @@ contains the total available logical CPUs available when LXD started.
 ## `vm_limits_cpu_pin_strategy`
 
 Adds a new {config:option}`instance-resource-limits:limits.cpu.pin_strategy` configuration option for virtual machines. This option controls the CPU pinning strategy. When set to `none`, CPU auto pinning is disabled. When set to `auto`, CPU auto pinning is enabled.
+
+## `unix_hotplug_ownership_inherit`
```

Comment:
should this be `unix_hotplug_ownership_inherit` to be consistent with existing `unix_device_hotplug` api extension?

Reply:
If you mean `unix_device_hotplug_ownership_inherit`, yes I think that's more consistent 👍